File: app/src/main/java/can/siempredelao/f1kotlin/backend/RaceTable.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RaceTable {

    public String     season;
    public String     round;
    public String     fastest;
    public String     position;
    public String     driverId;
    public String     constructorId;
    public String     grid;
    @SerializedName("Races")
    public List<Race> races;

}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/RaceTable.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

class RaceTable(val season: String,
                val round: String,
                val fastest: String,
                val position: String,
                val driverId: String,
                val constructorId: String,
                val grid: String,
                @SerializedName("Races") val races: List<Race>)
